include/Parser.h

```diff
@@ -171,8 +171,7 @@ namespace Parser {
 		Option{
 			RANGE, "-r", "",
 			{
-				"Set the range for the update rate of the",
-				"Matrix characters",
+				"Set the range for the update rate of the Matrix characters",
```

Comment:
This exceeds 80 characters when help is printed. I'm flexible on other things but help must fit inside a width of 80 characters.

Reply:
Okay, I hear that. Then it's probably best to reword this?

Reply:
How about something like `Set the range for the character update rate`? It's informative enough, and fits.

Reply:
Yup, that works. I that case also remove `Matrix` from the description of the color option.

Reply:
Done.